simplivity/resources/omnistack_clusters.py

```diff
@@ -116,3 +116,26 @@ def __init__(self, connection, resource_client, data):
         self.data = data
         self._connection = connection
         self._client = resource_client
+        self._clusters = OmnistackClusters(self._connection)
+
+    def __refresh(self):
+        """Updates the omnistack cluster data."""
+        resource_uri = "{}/{}".format(URL, self.data["id"])
+        self.data = self._client.do_get(resource_uri)['omnistack_cluster']
```

Comment:
use DATA_FILED

Reply:
Since Data Field has value 'omnistack_clusters', and we need 'omnistack_cluster' (there is a problem of extra "s"), that is the only reason we did not use DATA_FIELD variable

Reply:
We have a few options, (1) hard_coded, (2) use a RESPONSE_FIELD / DATA_FIELD within the class that has the field name ( i think in a different PR is was called OBJECT_TYPE, (3) use a ENUM Class to determine it what it should be based off the resource_uri

Should we go ahead and implement an ENUM Class to know what the response field name should be for each call?

Reply:
@svt-kmcdaniel, let's go with the OBJECT_TYPE/DATA_FIELD in the resource class so that it will be easy to write test cases also.

Reply:
Added OBJECT_TYPE field to address the issue.